src/Text/Pandoc/Logging.hs

```diff
@@ -76,6 +76,7 @@ data LogMessage =
   | InlineNotRendered Inline
   | BlockNotRendered Block
   | DocxParserWarning Text
+  | PptxParserWarning Text
```

Comment:
Since this is an API change, make sure your commit message clearly indicates this (if it doesn't already), so I can document it in the changelog.

Reply:
Ok, will do.